thunder/core/transforms.py

```diff
@@ -2293,6 +2300,8 @@ def iter_bound_symbols(bound_symbols):
     for symbol in bound_symbols:
         if symbol.sym.id in trace_interpreter_skip_list:
```

Comment:
We have one skip list here, `trace_interpreter_skip_list`, instead of adding a new one the existing list should be reused.

Reply:
Thanks. That's really helpful.

sounds like `is_constant_for_vjp` is actually what I wanted.  i.e. it skips the backward but keeps forward as is?

I don't think `trace_interpreter_skip_list` is the right list though. It's used in trace_interpreter.py as well, which is outside the context of grad. But since you pointed out `is_constant_for_vjp`, I think `trace_interpreter_skip_list` is the right list to add them. I'll refactor this.